lib/mayaUsd/ufe/UsdUIInfoHandler.cpp

```diff
@@ -71,6 +73,48 @@ bool UsdUIInfoHandler::treeViewCellInfo(const Ufe::SceneItem::Ptr& item, Ufe::Ce
 	return changed;
 }
 
+std::string UsdUIInfoHandler::treeViewIcon(const Ufe::SceneItem::Ptr& item) const
```

Comment:
Override this UFE method to provide an image for specific node types. Avoids the ancestor types call.